### PR TITLE
[Cosmos DB] az cosmosdb create/update: add --enable-public-network support

### PR DESCRIPTION
**Pull request by soeom**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

File: src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def load_arguments(self, _):
             c.argument('virtual_network_rules', nargs='+', validator=validate_virtual_network_rules, help='ACL\'s for virtual network')
             c.argument('enable_multiple_write_locations', arg_type=get_three_state_flag(), help="Enable Multiple Write Locations")
             c.argument('disable_key_based_metadata_write_access', arg_type=get_three_state_flag(), help="Disable write operations on metadata resources (databases, containers, throughput) via account keys")
+            c.argument('public_network_access', help="Enable or disable public network access to server.")
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py:67]: Could you please use enable_xxx like sql? 
https://github.com/Azure/azure-cli/blob/298994660f0fde6863cb45a7c3142141ed10f923/src/azure-cli/azure/cli/command_modules/sql/_params.py#L1069-L1074

**Comment by soeom** [comment]
@Juliehzl  Addressed your comment. Please take a look again

**Comment by soeom** [comment]
@Juliehzl Please help merge the PR.

**Comment by soeom** [comment]
@zhoxing-ms @Juliehzl @qianwens Can someone please help merge this PR? This is going on for too long.

**Pull request merged**